openssl/src/ec.rs

```diff
@@ -459,12 +459,35 @@ impl EcKeyBuilderRef {
     pub fn generate_key(&mut self) -> Result<&mut EcKeyBuilderRef, ErrorStack> {
         unsafe { cvt(ffi::EC_KEY_generate_key(self.as_ptr())).map(|_| self) }
     }
+
+    /// Sets the public key based on affine coordinates.
+    pub fn set_public_key_affine_coordinates(&mut self,
+                                             x: &mut BigNumRef,
```

Comment:
My checks seem to indicate that `x` and `y` aren't modified, at least not right now.

Unfortunately, the OpenSSL definition does not use `const` pointers on this function for `x` and `y`. I changed the Rust signature to reflect this.

n.b. I originally used `const` pointers in the extern and non-mutable references for `x` and `y` here but that led to failures in CI due to pointer mismatch.